src/lib/ppx_coda/versioned_util.ml

```diff
@@ -13,6 +13,18 @@ let check_modname ~loc name =
     Location.raise_errorf ~loc
       "Expected a module named Stable, but got a module named %s." name
 
+(* replace newlines in standard formatter with a space, so string is all on one line *)
```

Comment:
This will affect all other ppxs that dune links `ppx_coda` with, which may cause some weirdness, and makes me pretty uncomfortable. Can we create a new formatter instead? (Shadowing `Format` as a library in `ppx_coda` that overrides the `print_*` and `std_formatter` definitions should do this fairly transparently.)

Reply:
oh, I'd forgotten this code uses side effects.

will fix.